glue/bin/run-vnc

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -ex
+
+mkdir -p "$XDG_RUNTIME_DIR" -m 700
+mkdir -p "$XDG_CACHE_HOME"
+
+DISPLAY_NO=66
+DISPLAY=:${DISPLAY_NO}
+x11_auth_file=$(mktemp --tmpdir="${XDG_RUNTIME_DIR}")
+
+# If there's already a compositor for WAYLAND_DISPLAY choose another
+if [ -O "${XDG_RUNTIME_DIR}/${WAYLAND_DISPLAY}" ]; then
+  port=0
+  while [ -e "${XDG_RUNTIME_DIR}/wayland-${port}" ]; do
+    let port+=1
+  done
+  WAYLAND_DISPLAY=wayland-${port}
+fi
+
+xvfb-run --auth-file=${x11_auth_file} --server-num=${DISPLAY_NO} "$@"&
+xvfb_pid=$!
+
+trap "kill $xvfb_pid" INT TERM
+
+until [ -O "${XDG_RUNTIME_DIR}/${WAYLAND_DISPLAY}" ]
+do
+  inotifywait -qq --timeout 5 --event create $(dirname "${XDG_RUNTIME_DIR}/${WAYLAND_DISPLAY}")
+done
+
+env -u WAYLAND_DISPLAY x11vnc -display ${DISPLAY} -bg -nopw -listen localhost -xkb -auth ${x11_auth_file} -ncache 10
```

Comment:
Why not listen on 0? You could require a password to be set for security, but it would reduce the need for the ssh tunnel?

Reply:
OK.